user/src/main/java/org/dicadeveloper/weplantaforest/projects/Price.java

```diff
@@ -37,6 +41,7 @@
     private ScontoType scontoType = ScontoType.NONE;
 
     @Column(name = "_amount", precision = 7, scale = 2)
+    @JsonView({Views.Project.class})

```

Comment:
Does this mean that the view project contains also an price?
Where is this view defined?